spec/built-in-modules/meta.md

```diff
@@ -144,6 +162,31 @@ This function is also available as a global function named `get-function()`.
   * Return [`use`'s module][]'s function named `$name`, or throw an error if no
     such function exists.
 
+### `get-mixin()`
+
+```
+get-mixin($name, $module: null)
+```
+
+* If `$name` is not a string, throw an error.
+
+* If `$module` is null:
+
+  * Return the result of [resolving a mixin] named `$name`. If this returns
```

Comment:
the `resolving a mixin` method seems to make it like `get-mixin('foo.bar')` should work and return the `bar` mixin from the namespace `foo`, but that doesn't seem to be the case to how it's implemented. 

Should this clarify that it should be the name for a mixin declared in the current namespace?